health/db-sync-state.js

```diff
@@ -37,20 +37,7 @@ module.exports = exports = new (class DBSyncStateCheck extends nHealthCheck {
 })({
 	businessImpact: 'The Syndication database\'s computed tables are not in-sync with the base table data and, as such, may be showing incorrect data to users.',
 	name: 'Syndication database data integrity',
-/*eslint-disable*/
-	panicGuide: `To force a reload of all the computed tables, run:
-
-\`\`\`
-	~$ psql --username \${PRODUCTION_DATABASE_USER_NAME}
-			--password \${PRODUCTION_DATABASE_PASSWORD}
-			--host \${PRODUCTION_DATABASE_HOST}
-			--port \${PRODUCTION_DATABASE_PORT}
-			--dbname \${PRODUCTION_DATABASE_NAME}
-			--command 'select syndication.reload_all();'
-\`\`\`
-
-Substituting the above \`\${PRODUCTION_DATABASE_*}\` placeholders with the correct values from vault.`,
-/*eslint-enable*/
+	panicGuide: 'you can sync them with the repo as follows: 1. ngage get-config --env prod 2. make sync',
```

Comment:
This should explain that you should clone and cd into the repo first I think

Reply:
@leggsimon good point, let's not merge this PR... I think maybe it'll be nice to have a check early in `n-gage` to check whether .env is production, and prompt the user "is this what you want to load?"